### PR TITLE
Milestone PR :- Week 1, 2 and 3

### PR DESCRIPTION
**Pull request by H4R5H1T-007**
## Project

Document Uploader

### Changes

Added Week 1, 2 and 3 weekly milestone updates.

### CODE REVIEW

**Review by wasup-yash — APPROVED** [review]

File: docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md [hunk 1]
```diff
@@ -4,13 +4,18 @@ author: Harshit Gupta
...
+<!-- - [ ] Give the description about Milestone 4 -->
 
 ## Screenshots / Videos 
 
 ## Contributions
+- [Here's the link](https://colab.research.google.com/drive/1AgRUKep8juyyJcOik9kY1PFnkc8OCxhe?usp=sharing) to the google colab notebook used to create a benchmark and test it against Jugalbandi API
 
 ## Learnings
+- Looked out Mertics for Text segmentation and how to calculate them.
+- Learned more about text segmentation models.
```
Comment (Apoorvgarg-creator) [inline docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md:20]: Hey, It would be better, if you can briefly explain all the learning.

**Review by Apoorvgarg-creator — CHANGES_REQUESTED** [review]

File: docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md (hunk 1 above)
Comment (wasup-yash) [reply docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md:20]: is it not personalized as of with the project scope? or are there any guidelines that I am not aware of?

Comment (H4R5H1T-007) [reply docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md:20]: Can I add links of the blogs which I read instead for anyone to read? @Apoorvgarg-creator. I have added them into the tickets might as well add it here.

Comment (H4R5H1T-007) [reply docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md]: Done. @Apoorvgarg-creator @wasup-yash

**Review by wasup-yash — APPROVED** [review]

File: docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md (hunk 1 above)
Comment (Apoorvgarg-creator) [reply docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md]: Hey, @H4R5H1T-007 Sorry for not replying, I was busy ! Nice Updates

Comment (H4R5H1T-007) [reply docs/2023/AI Tools/Document Uploader/updates/2023-07-07.md]: > Hey, @H4R5H1T-007 Sorry for not replying, I was busy ! Nice Updates

So if everything is okay then please merge it now. @Apoorvgarg-creator @wasup-yash

**Pull request merged**